public/style/sass/_report.scss

```diff
@@ -15,21 +9,67 @@
   z-index: 1;
 }
 
-.form-button {
-  outline:none;
+.input-label,
+.type-label,
+.type-wrap {
+  font: 15px $default;
+  font-weight: 500;
+  margin-bottom: 2px;
+}
+
+.button {
+  outline: none;
   width: 100%;
   height: 40px;
-  border: 1px solid #ccc;
-  background-color: #eee;
   font: 15px $default;
   cursor: pointer;
   padding: 0.5em;
 }
 
-.form-button:not(:last-child) {
+.form-button {
+  border: 1px solid #ccc;
+  background-color: #eee;
+}
```

Comment:
True for the most part except now the type checkbox field, which I've grouped with .button. However the checkbox fieldset needs to have different spacing. So I think we need to keep these separate. 

I just need to keep a height rule for .form-button separate from .button 

See what you think